.vscode/snowplow-docs.code-snippets

```diff
@@ -0,0 +1,151 @@
+{
+	// Place your Documentation workspace snippets here. Each snippet is defined under a snippet name and has a scope, prefix, body and
+	// description. Add comma separated ids of the languages where the snippet is applicable in the scope field. If scope
+	// is left empty or omitted, the snippet gets applied to all languages. The prefix is what is
+	// used to trigger the snippet and the body will be expanded and inserted. Possible variables are:
+	// $1, $2 for tab stops, $0 for the final cursor position, and ${1:label}, ${2:another} for placeholders.
+	// Placeholders with the same ids are connected.
+	// Example:
+	// "Print to console": {
+	// 	"scope": "javascript,typescript",
+	// 	"prefix": "log",
+	// 	"body": [
+	// 		"console.log('$1');",
+	// 		"$2"
+	// 	],
+	// 	"description": "Log output to console"
+	// }
+	"Header": {
+			"scope": "markdown",
+			"prefix": "head",
+			"body": [
+				"---",
+				"title: \"$1\"",
+				"sidebar_position: 10",
```

Comment:
Would change this to `1` — we know know that Docusaurus supports floating point positions, so there is no need to do multiples of 10

Reply:
I understand they support it, I just don't like it 😆 
I'll make it a tab stop, compromise.